delivery_package_number_theoretical/models/delivery_carrier.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class DeliveryCarrier(models.Model):
+
+    _inherit = "delivery.carrier"
+
+    maximum_weight_per_package = fields.Float(string="Maximum weight per package")
```

Comment:
@glitchov As it's done for the product form, you should display the uom name of the uom defined for volumes...

```python
    weight_uom_name = fields.Char(string='Weight unit of measure label', compute='_compute_weight_uom_name')

    def _compute_weight_uom_name(self):
        self.weight_uom_name = self.env["product.template"]._get_weight_uom_name_from_ir_config_parameter()
```